code/controllers/subsystem/events.dm

```diff
@@ -246,6 +271,16 @@ SUBSYSTEM_DEF(events)
 		var/datum/event_meta/EM = E.event_meta
 		log_and_message_admins("has stopped the [GLOB.severity_to_string[EM.severity]] event '[EM.name]'.")
 		E.kill()
+	else if(href_list["add_modifier"])
```

Comment:
Sidenote, can you put a `check_rights(R_EVENT)` at the start of this proc?

If someone manages to get the UID of this controller for whatever reason they could send arbitrary hrefs to it